src/Grok.Net.Tests/Patterns/grok-custom-patterns

```diff
@@ -1 +1,3 @@
-﻿ZIPCODE [1-9]{1}[0-9]{2}\s{0,1}[0-9]{3}
\ No newline at end of file
+﻿ZIPCODE [1-9]{1}[0-9]{2}\s{0,1}[0-9]{3}
```

Comment:
Hi @GioMaz, did you try to run all tests after this change? It seems will break all other tests

Reply:
All the unit tests work for me, could you please send me more information about this?